afqbrowser/site/client/js/save-settings.js

```diff
@@ -168,3 +168,15 @@ afqb.table.restoreRowSelection = function () {
         }
     });
 };
+
+$('#launch-binder').click(function (e) {
+    e.preventDefault();
+    // Parse the URL
+    var uri = new URI(location.href);
+    var user = uri.hostname();
```

Comment:
Oops, forgot to extract the github user/org name from the hostname. Should be something like
```
var user = user.hostname().replace('.github.io/', '');
```